apps/test-suite/tests/Location.js

```diff
@@ -337,13 +342,15 @@ export async function test(t) {
 
     describeWithPermissions('Location.watchPositionAsync()', () => {
       t.it('gets a result of the correct shape', async () => {
-        await new Promise(async (resolve, reject) => {
-          const subscriber = await Location.watchPositionAsync({}, (location) => {
-            testLocationShape(location);
-            subscriber.remove();
-            resolve();
+        let subscriber;
+        const location = await new Promise(async (resolve) => {
```

Comment:
this promise definition seems unneeded if you await on it instantly, but I know this is old code you're just moving around